src/payment/__snapshots__/PaymentPage.test.jsx.snap

```diff
@@ -6438,37 +6444,19 @@ exports[`<PaymentPage /> Renders correctly in various states should render the b
             </p>
           </div>
           <form
-            className="summary-row d-flex align-items-end"
+            className="d-flex align-items-center mb-3"
             onSubmit={[Function]}
           >
-            <div
-              className="form-group mb-0 mr-2"
-            >
-              <label
-                className="h6 d-block"
-                htmlFor="couponField"
-              >
-                Add coupon code (optional)
```

Comment:
Did the default state change from Adding a Coupon mode to a Removal of Coupon mode?  You may want to double-check why the snapshot test changed in this way.

Reply:
@albemarle I found the reason for this, and since fixed it. Our mock stores had coupon data in two places and they didn't match.